packages/swapper/src/swappers/RelaySwapper/endpoints.ts

```diff
@@ -32,9 +32,14 @@ import { chainIdToRelayChainId } from './constant'
 import { getTradeQuote } from './getTradeQuote/getTradeQuote'
 import { getTradeRate } from './getTradeRate/getTradeRate'
 import { getLatestRelayStatusMessage } from './utils/getLatestRelayStatusMessage'
+import { notifyTransactionIndexing } from './utils/notifyTransactionIndexing'
 import { relayService } from './utils/relayService'
 import type { RelayStatus } from './utils/types'
 
+// Keep track of the trades we already notified the relay indexer about
+const txIndexingMap: Map<string, boolean> = new Map()
+const txByQuoteIdMap: Map<string, EvmTransactionRequest> = new Map()
```

Comment:
We are only supporting EVM chains for that thing as we don't support wrap/unwrap for Solana on both Jupiter and relay side, wsol is the same assetId on jupiter side, it was the main reason why we couldn't support it